lib/pbench/server/api/resources/query_apis/controllers_list.py

```diff
@@ -142,3 +151,23 @@ def postprocess(self, es_json: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
             controllers.append(c)
         # construct response object
         return jsonify(controllers)
+
+
+class PublicControllersList(ControllersList):
+    """
+        Get the names of public controllers within a date range.
+    """
+
+    def __init__(self, config: PbenchServerConfig, logger: Logger):
+        ElasticBase.__init__(
+            config,
+            logger,
+            Schema(
+                Parameter("start", ParamType.DATE, required=True),
+                Parameter("end", ParamType.DATE, required=True),
+            ),
+        )
+
+    def assemble(self, json_data: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
+        json_data["user"] = None
```

Comment:
Why is the user added here as `None`?  Can you add a comment explaining why we have to do this?

Reply:
Same comments for the other modules where this occurs.

Reply:
Fixed and added the comment